docs: consolidate cyberattack wiki articles and de-draft supply-chain (#1154)
1712n/dn-institute: content/research/cyberattacks/wiki/governance-attacks.md

---
title: Governance Attacks
bookToc: true
---

Governance attacks exploit the fact that many DAOs treat token ownership as control. If an attacker can cheaply accumulate voting power, borrow it through a flash loan, or exploit low voter participation, they can pass proposals that transfer treasury assets, mint tokens, or rewrite protocol rules.

## The Mechanism

In a standard DAO, governance power is proportional to token ownership or delegated voting weight. That model becomes fragile when:

1. **Flash loans** let an attacker borrow enough capital to dominate a vote in one transaction, as seen in [Beanstalk](https://dn.institute/attacks/posts/2022-04-17-Beanstalk/).
2. **Low participation** allows hostile proposals to pass when honest voters are absent or disengaged.
3. **Thin governance-token liquidity** lets an attacker buy effective control cheaply and then extract more value than the acquisition cost.

> ## Famous Case Studies

Reviewer: Use the same section name as the bridge and frontend pages so all five articles match.

```suggestion
## Case Studies
```

### 1. Beanstalk Farms (April 2022)
- **Loss:** Approximately [$182 million](https://dn.institute/attacks/posts/2022-04-17-Beanstalk/).
- **Mechanism:** Flash-loan governance attack.

> - **Details:** The attacker used a flash loan of nearly [$1 billion](https://dn.institute/attacks/posts/2022-04-17-Beanstalk/) in assets to gain a 67% governance supermajority, then executed proposal BIP-18 to transfer protocol funds.

Reviewer: Two concrete facts worth adding: BIP-19 was the Ukraine donation proposal paired with the malicious BIP-18 as cover, and the attacker netted about 80 million of the 182 million removed.

```suggestion
- **Details:** The attacker used a flash loan of nearly [$1 billion](https://dn.institute/attacks/posts/2022-04-17-Beanstalk/) in assets to gain a 67% governance supermajority, then used the `emergencyCommit` path to execute the malicious proposal BIP-18 and drain protocol funds; a paired proposal, BIP-19, posed as a Ukraine donation to disguise intent. The attacker netted [about $80 million](https://medium.com/immunefi/hack-analysis-beanstalk-governance-attack-april-2022-f42788fc821e) of the roughly $182 million removed.
```

- **Speed:** Slower than flash-loan exploits because it depended on the governance timetable, but still effective because almost nobody showed up to stop it.

### 3. True Seigniorage Dollar (TSD) (March 2021)
- **Loss:** Reporting on the incident focused less on a stable realized dollar total than on the fact that the attacker minted [11.8 billion TSD](https://tokenpost.com/news/investing/7404) and collapsed the token's market value.
- **Mechanism:** Governance arbitrage via cheaply accumulated voting power.
- **Details:** In the TSD takeover, the attacker accumulated enough governance influence to pass a self-serving proposal, minted [11.8 billion TSD](https://tokenpost.com/news/investing/7404), and dumped the newly created supply on PancakeSwap. This was a TSD incident, not Empty Set Dollar (ESD), and it illustrates how thin-liquidity governance systems can be looted without flash loans.
- **Speed:** The accumulation phase took longer than Beanstalk, but the destructive payout and market collapse followed quickly once governance control was obtained.

> ## Prevention and Mitigation

Reviewer: Use Mitigations to match the heading on the other pages.

```suggestion
## Mitigations
```
